src/proxies/stationsapi.js

```diff
@@ -0,0 +1,79 @@
+const platformClient = require('purecloud-platform-client-v2');
+const retry = require('@lifeomic/attempt').retry;
+const {sleep} = require('@lifeomic/attempt');
+
+/*
+    When you create a web rtc phone you do not automatically associate it with the user.
+    The getStationByWebRtcUserId() will look up the station for the web rtc phone and
+    return it so that we can then assign the user to it.
+*/
+const getStationByWebRtcUserId = async (userId) => {
+  let opts = {
+    webRtcUserId: userId,
+  };
+
+  let apiInstance = new platformClient.StationsApi();
+
+  try {
+    const results = await retry(
+      async (context) => {
+        const stations = await apiInstance.getStations(opts);
+        /*If we cant find a station then throw an exception.  This will trigger a retry*/
+        if (stations.entities.length === 0) {
+          console.log(`No station found retrying`);
+          throw 'No station found, retrying';
```

Comment:
Don't throw strings. Throw errors instead.
```suggestion
          throw new Error('No station found, retrying');
```

Reply:
You probably don't want to ultimately throw an error saying you're retrying when you've stopped retrying.

Reply:
Come up with a better error.